pkg/apis/contrail/v1alpha1/kubemanager_types.go

```diff
@@ -184,7 +187,13 @@ func (c *Kubemanager) InstanceConfiguration(request reconcile.Request,
 		}
 		kubemanagerConfig.ServiceSubnets = serviceSubnets
 	}
-
+	keystoneAuth := &KubemanagerAuthParameters{}
+	if configNodesInformation.AuthMode == AuthenticationModeKeystone {
+		err := c.AuthParameters(keystoneAuth, client)
```

Comment:
Authentication parameters should be injected from manager.go 

Here we should for keystone to become active https://github.com/Juniper/contrail-operator/blob/ec8d40e3323efdf7b6f89a12957d7479b419bcbd/pkg/controller/manager/manager_controller.go#L571

and here we should create a configuration https://github.com/Juniper/contrail-operator/blob/ec8d40e3323efdf7b6f89a12957d7479b419bcbd/pkg/controller/manager/manager_controller.go#L1066